### PR TITLE
Ft login enhanced

### PR DESCRIPTION
**Pull request by Balinda21**
### What does this PR do?
- Adds the `isPasswordExpired` field to the data returned upon successful login.
### Description of Task to be completed?
- Modify the backend login functionality to include `isPasswordExpired` in the response.
- Ensure that the field accurately reflects the password expiration status.
### How should this be manually tested?
1. **Setup**:
   - Pull the latest changes from this branch.
   - Run the application.

2. **Steps**:
   - Use the login endpoint with valid credentials.
   - Verify that the response includes the `isPasswordExpired` field.
   - Test with accounts having different password expiration statuses.
### Any background context you want to provide?
- The addition of the `isPasswordExpired` field helps in notifying users if they need to update their password.

### CODE REVIEW

**Review by bahati10 — APPROVED** [review]
Great Job on this change

**Review by Ishimwe7 — APPROVED** [review]
Thank you for this necessary changes @Balinda21

File: src/controllers/userController/loginUser.ts [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
+      const isPasswordValid = await bcrypt.compare(password, user.password);
+      if (!isPasswordValid) {
+        res.status(401).json({ error: 'Invalid credentials' });
+        return;
+      }
+
+      const token = jwt.sign(
+        { userId: user.id },
+        process.env.VERIFICATION_JWT_SECRET as string,
+        { expiresIn: '1h' }
```
Comment (kirengamartial) [inline src/controllers/userController/loginUser.ts:27]: extend the expiration date

**Review by kirengamartial — APPROVED** [review]
good job morris

**Pull request merged**